spec/requests/userscontroller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    before(:each) do
+      get '/'
+    end
+
+    it 'GET request return correct response' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'response body includes correct text' do
+      expect(response.body).to include('list of users')
+    end
+
+    it 'renders a correct template' do
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe 'Get #show' do
+    before(:each) do
+      get '/users/798'
+    end
+
+    it 'GET request return correct response' do
+      expect(response).to have_http_status(:ok)
+      expect(response).to render_template(:show)
+      expect(response.body).to include('user profile')
+    end
+  end
+end
```

Comment:
- [ ] As already mentioned above, please make sure that all these tests are passing.

Reply:
![Screenshot from 2022-04-30 10-22-15](https://user-images.githubusercontent.com/52419124/166100063-a64177dd-6490-4db8-9ff6-15f4e821b6c2.png)